daemons/fenced/fenced_remote.c

```diff
@@ -570,8 +578,13 @@ remote_op_done(remote_fencing_op_t * op, xmlNode * data, int rc, int dup)
 
     handle_local_reply_and_notify(op, data, rc);
 
-    if (dup == FALSE) {
-        handle_duplicates(op, data, rc);
+    if (!dup) {
+        pcmk__action_result_t result = PCMK__UNKNOWN_RESULT;
+
+        pcmk__set_result(&result,
+                         ((rc == pcmk_ok)? CRM_EX_OK : CRM_EX_ERROR),
+                         stonith__legacy2status(rc), NULL);
+        finalize_op_duplicates(op, data, &result);
```

Comment:
should we reset result either here or in finalize_op_duplicates?

Reply:
no need, exit reason is NULL here